### DISCUSSION TITLE
No way to add dataset

### DISCUSSION THREAD

**Discussion by jules-o** [broadinstitute/seqr#1950] [Q&A]
Hi,

Using local install via docker, just upgraded and now when I click Edit Datasets, instead of the form to give the elasticsearch index I'm getting a popup to add IGV paths.

![Screenshot from 2021-05-10 23-27-54](https://user-images.githubusercontent.com/41348829/117732519-f02d6280-b1e7-11eb-96f9-497629de4061.png)

Am I doing something wrong or is this a bug?

**Comment by hanars** [comment]
we've recently made permissions in seqr more granular. To add elasticsearch indices you need to have the is_staff permission set for your user

**Comment by jules-o** [comment]
Hi Hana, sorry to bother you again but how do I set the is_staff permission for a user? I'm logged in as a Data Manager, looking at the Data Management>Users page, where I can see the user but can't modify anything. I tried to look in Kibana but get: 
Error: Unable to connect to Kibana HTTPConnectionPool(host='kibana', port=5601): Max retries exceeded with url: /app/kibana (Caused by NewConnectionError(': Failed to establish a new connection: [Errno -2] Name or service not known'))

I'm also now getting an error when I try to variant search: 
`relation "reference_data_geneinfo" does not exist LINE 1: ...id", "reference_data_geneinfo"."gene_symbol" FROM "reference... ^ `
I've run migrations and updated gene-level reference datasets via docker exec, not sure what else to try.

**Comment by hanars** [comment]
you need to use the /admin page to directly modify the DB

**Comment by hanars** [comment]
and the second issue you mention is definitely a migration issue. Make sure you run migrate with the
 `--database=reference_data_db` flag to apply migrations to that database

**Comment by jules-o** [comment]
Thanks Hana, got the user updated now and fixed the migrations. It's the wonderful seqr that I know and love again :)
